iree/runtime/setup.py

```diff
@@ -0,0 +1,379 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# This builds just the runtime API and is relatively quick to build.
+# To install:
+#   pip install .
+# To build a wheel:
+#   pip wheel .
+#
+# It is recommended to build with Ninja and ccache. To do so, set environment
+# variables by prefixing to above invocations:
+#   CMAKE_C_COMPILER_LAUNCHER=ccache CMAKE_CXX_COMPILER_LAUNCHER=ccache
```

Comment:
Up to you, but I'd recommend lld here too :) (although I see CI doesn't use it either)

Reply:
I thought lld is default now ?

Reply:
Let me land vanilla stuff and then we can play and extend. In general, it is common practice for ad-hoc Python packages to build with system defaults.